app.js

```diff
@@ -0,0 +1,29 @@
+(function (require, process, console) {
```

Comment:
I don't think we need to wrap the files in IIFEs. The purpose of the IIFE is to prevent the code from declaring variables on the global scope. In node, each file (or module) is encapsulated in its own scope, so the code won't affect the global scope.